launch/tier4_localization_launch/launch/pose_twist_estimator/pose_twist_estimator.launch.xml

```diff
@@ -112,4 +112,31 @@
       <arg name="use_eagleye_twist" value="true"/>
     </include>
   </group>
+
+  <!-- AR Tag Based Localizer with NDT Launch (as pose estimator) -->
+  <group if="$(eval &quot;'$(var pose_source)'=='ndt_and_ar_tag_based_localizer'&quot;)">
```

Comment:
Thank you for your suggestion.
I prefer full names to abbreviations.
Please leave it as it is.

I think that it will be solved naturally when multi-pose_source input is implemented.